git/commands.md

```diff
@@ -11,3 +11,19 @@ git init
 from the command line at the folder.
 
 After running the command respective git's technical files would be created.
+
+##### Add file/directory
+Adds files in the to the staging area for Git. Before a file is available to commit to a repository, the file needs to be added to the Git index (staging area). There are a few different ways to use git add, by adding entire directories, specific files, or all unstaged files.
```

Comment:
The line has minor grammar issues: Adds files to the Git's staging area. Before committing a file to a repository, should be added to the Git index (staging area). There are a few different ways to use `git add`, by adding entire directories, specific files, or whole bunch of unstaged files.